node/core/provisioner/src/metrics.rs

```diff
@@ -22,8 +22,11 @@ struct MetricsInner {
 	inherent_data_requests: prometheus::CounterVec<prometheus::U64>,
 	request_inherent_data: prometheus::Histogram,
 	provisionable_data: prometheus::Histogram,
+
+	/// The dispute_statement.* metrics trak how many disputes/votes the runtime will have to process. It will count
+	/// all recent statements meaning every dispute from last sessions: 10 min on Rococo, 60 min on Kusama and
+	/// 4 hours on Polkadot. The metrics are updated only when the node authors block, so values vary across nodes.
```

Comment:
`cargo spellcheck fix --checker=hunspell $(git diff --only-names $(git merge-base origin/master)..HEAD)` is a nice helper, (or even a good pre-commit hook when removing the `fix`)